### PR TITLE
Reduce SHA1s tagged on images to 7 characters. TE-1717

### PR DESCRIPTION
**Pull request by macdiesel**
@edx/pipeline-team @jibsheet PTAL.

### CODE REVIEW

File: playbooks/continuous_delivery/create_ami.yml [hunk 1]
```diff
@@ -73,9 +73,9 @@
       description: "AMI built via edX continuous delivery pipeline - Ansible version: {{ ansible_version }}"
       # used a JSON object here as there is a string interpolation in the keys.
       tags: "{
-                'version:{{ play }}':'{{ app_repo }} {{ app_version }}',
-                'version:configuration':'{{ configuration_repo }} {{ configuration_version }}',
-                'version:configuration_secure':'{{ configuration_secure_repo }} {{ configuration_secure_version }}',
+                'version:{{ play }}':'{{ app_repo }} {{ app_version[:7] }}',
```
Comment (doctoryes) [inline playbooks/continuous_delivery/create_ami.yml:76]: Should the `app_version` used above as `name` also only use the first seven characters just to be consistent?

Comment (macdiesel) [reply playbooks/continuous_delivery/create_ami.yml:76]: done

**Comment by doctoryes** [comment]
👍

**Comment by jibsheet** [comment]
👍

**Pull request merged**